packages/babel-parser/src/parser/util.js

```diff
@@ -192,29 +190,6 @@ export default class UtilParser extends Tokenizer {
     }
   }
 
-  strictDirective(start: number): boolean {
```

Comment:
This is now dead code.

Reply:
<details>
<summary> :warning: Spoiler :warning: </summary>

This is probably because before this PR, we first checked if there was a `use strict` directive _and then_ tokenized the invalid literals.
Now, we first tokenize the invalid literals, then set `this.state.strict` to `true`, and in `parseBlockOrModuleBlockBody` only throw for the first invalid one. It should be fixed there.

</details>